bears/r/FormatRBear.py

```diff
@@ -5,13 +5,42 @@
 class FormatRBear(Lint, LocalBear):
     executable = "Rscript"
     arguments = ("-e 'library(formatR)' "
-                 "-e 'formatR::tidy_source(\"{filename}\")'")
+                 "-e 'formatR::tidy_source(\"{filename}\"")
     prerequisite_command = ['Rscript', '-e', "'library(formatR)'"]
     prerequisite_fail_msg = "Please install formatR for this bear to work."
     diff_message = "Formatting can be improved."
     gives_corrected = True
     LANGUAGES = "R"
 
+    def set_tidy_arguments(arguments):
+        coment = input("formatR.Comment: ")
```

Comment:
We don't normally use `input` as we don't want coala to stop here.

The user can specify the configs in the `.coafile` and so on too !
Hence, you need to give it in the arguments on `run` where coala will automatically send configs given in the config file.

Check how https://github.com/coala-analyzer/coala-bears/blob/77279d3ca7fff4728c2b75f5e203c20b1e9836dd/bears/go/GoErrCheckBear.py does it.
Also, read the Writing bears tutorial in the docs http://coala.readthedocs.io/en/latest/